devicon.json

```diff
@@ -5676,6 +6287,9 @@
     },
     {
         "name": "tomcat",
+        "altnames": [
+            "tomcat"
```

Comment:
Altname is the same as name.

Reply:
3c5bf839850823623296b0f0a222ba65aff6386c